jdaviz/configs/imviz/tests/test_catalogs.py

```diff
@@ -91,7 +91,7 @@ def test_plugin_image_with_result(self, imviz_helper, tmp_path):
         self.imviz = imviz_helper
 
         catalogs_plugin = self.imviz.app.get_tray_item_from_name('imviz-catalogs')
-        catalogs_plugin._obj.plugin_opened = True
+        catalogs_plugin.plugin_opened = True
```

Comment:
why this change?  `plugin_opened` isn't exposed through the user API afaik.  If you want to use public API, use `with plugin.as_active():` instead.

Reply:
It was failing the test. I removed the `_obj` to be consistent with the rest of this test module. The way this plugin object is accessed, I don't think it goes through the public API? I think these tests were written before public API concept was a thing.

Reply:
ah, it uses `get_tray_item_from_name`, ok, nevermind then!